includes/definitions/teradici-pcoip.yaml

```diff
@@ -0,0 +1,11 @@
+os: teradici-pcoip
+text: 'Teradici PCoIP'
+type: workstation
```

Comment:
I see lines that point to an icon for device types, i.e. `$config['device_types'][$i]['icon'] = 'server.png';`
Where are these stored? I could not find them